pytx/pytx/batch.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+from access_token import get_access_token
+from request import Broker
+
+from vocabulary import Batch as b
+from vocabulary import ThreatExchange as t
+from errors import (
+    pytxFetchError
+)
+
+
+class Batch(object):
+
+    """
+    Class for making Batch requests to the API.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initialized the object.
+        """
+
+    @classmethod
```

Comment:
You are correct. In deciding between `@staticmethod` and `@classmethod`, I went with the latter because if we decide to add any default attributes to the `Batch()` class I wanted to be able to start using them where needed without having to go back and adjust all of the methods. Granted right now we don't have a ton of code, but I was thinking long-term :)